lib/core/src/lib/form/components/widgets/form.theme.scss

```diff
@@ -7,8 +7,7 @@ ul > li > form-field > .adf-focus {
 
 .adf {
     &-error-container {
-        height: 20px;
-        margin-top: -12px;
+        margin-top: -8px;
```

Comment:
have u checked this change how will affect apps ?

Reply:
reverted